UI/CUIControls.cpp

```diff
@@ -1430,29 +1430,40 @@ namespace {
     // row type used in the SpeciesSelector
     struct SpeciesRow : public GG::ListBox::Row {
         SpeciesRow(const Species* species, GG::X w, GG::Y h) :
-            GG::ListBox::Row(w, h, "", GG::ALIGN_VCENTER, 0)
+                   GG::ListBox::Row(w, h, "", GG::ALIGN_VCENTER, 0)
```

Comment:
GG::ListBox::Row(...) is the base class constructor, not a parameter to SpeciesRow, so should only be indented 4 spaces, not to the (